src/lumigo_tracer/auto_instrument_handler.py

```diff
@@ -26,11 +26,21 @@ def _handler(*args, **kwargs):
     handler_module = ""
     try:
         handler_module, unit_name = parse_handler()
-        original_handler = getattr(importlib.import_module(handler_module), unit_name)
-    except (ImportError, AttributeError):
+        original_module = importlib.import_module(handler_module)
+    except ImportError as e:
         raise ImportError(
-            f"Unable to import module '{handler_module}': No module named '{handler_module}'"
-        )
+            f"Runtime.ImportModuleError: Unable to import module '{handler_module}': {str(e)}"
```

Comment:
Here you even add of original exception in the text. Don't you prefer to do `from e` instead?

Reply:
I wanted to copy AWS's bootstrap default behavior: 
![image](https://user-images.githubusercontent.com/47108628/103623770-38d6c580-4f41-11eb-9aa6-7bbfa83d0f13.png)